base/strings/string.jl

```diff
@@ -438,3 +449,25 @@ function repeat(s::String, r::Integer)
     end
     return out
 end
+
+"""
+    repeat(c::Char, r::Integer) -> String
```

Comment:
Why don't we use Julia syntax `repeat(c::Char, r::Integer)::String` for this sort of thing?